src/crypto/algorithms/base.ts

```diff
@@ -66,18 +66,16 @@ export abstract class EncryptionAlgorithm {
     protected readonly crypto: Crypto;
     protected readonly olmDevice: OlmDevice;
     protected readonly baseApis: MatrixClient;
-    protected readonly roomId?: string;
```

Comment:
This feels like it's snuck in from a separate PR?

Reply:
It did not, without it TSC in es2022 mode on downstream layers complains that you cannot override a field definition in a parent class without using `declare` syntax